lib/eventboss/development_mode.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Eventboss
+  module DevelopmentMode
+    extend Logging
+
+    class << self
+      def setup_infrastructure(queues)
+        sns_client = Eventboss.configuration.sns_client
+        sqs_client = Eventboss.configuration.sqs_client
+
+        queues.each do |queue, listener|
+          topic_name = Eventboss::Topic.build_name(**listener.options)
+          logger.info('development-mode') { "Creating topic #{topic_name}..." }
+          topic = sns_client.create_topic(name: topic_name)
```

Comment:
Will it work if the queue already exists?

Reply:
It will, no need to check beforehand.